### ISSUE TITLE
OIDC core mismatch - Error response on unmet essential claim requirements

### ISSUE THREAD

**Issue by Razumain** [oidc-sweden/specifications#44]
Section 4.4 states that an OP MUST respond with an error response if a requested claim marked as "essential" can't be provided.

However, this seems to be in conflict with OIDC core section 5.5.1 stating:

"_By requesting Claims as Essential Claims, the RP indicates to the End-User that releasing these Claims will ensure a smooth authorization for the specific task requested by the End-User. Note that even if the Claims are not available because the End-User did not authorize their release or they are not present, **the Authorization Server MUST NOT generate an error** when Claims are not returned, whether they are Essential or Voluntary_"

In other words. The OP MUST always return what it can deliver and leave it to the RP to decide whether that response is useful or not.

**Comment by martin-lindstrom** [comment]
Will fix this to comply with the core spec.
